### PR TITLE
SQL-566: Create very rough outline of script for starting local ADL against local mongod

### PR DESCRIPTION
**Pull request by mattChiaravalloti**
@bucaojit This is the bare-bones script. Feel free to just use it as a jumping off point!

### CODE REVIEW

File: resources/run_adl.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# 1. set git to use ssh instead of https
+git config --global url.git@github.com:.insteadOf https://github.com/
+
+# 2. clone the mongohouse repo
+# TODO - where to put it
+git clone git@github.com:10gen/mongohouse.git
+cd mongohouse
+
```
Comment (bucaojit) [inline resources/run_adl.sh:8]: Spawned a new host based on the evergreen run: ubuntu-1804.   The preinstalled go version was 1.7.  Will add suggestions to help install on that variant.
 ```suggestion

export GOVERSION=$(curl https://go.dev/VERSION?m=text)
wget "https://dl.google.com/go/${GOVERSION}.linux-amd64.tar.gz"
sudo rm -rf /opt/go && sudo tar -C /opt -xzf ${GOVERSION}.linux-amd64.tar.gz

```

File: resources/run_adl.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# 3. download external dependencies
+# TODO - make sure this works on evergreen
+go run cmd/buildscript/build.go tools:download:mqlrun
+
+# TODO - make sure this works on evergreen, also how to do this platform-specific
+go run cmd/buildscript/build.go tools:download:mongosql
```
Comment (bucaojit) [inline resources/run_adl.sh:14]: This command is failing with the following error.  I'll investigate:
```
$ go run cmd/buildscript/build.go tools:download:mongosql
START  | tools:download:mongosql
       | download:  -> artifacts
FAIL   | tools:download:mongosql in 131.295µs
       | failed issuing GET request: Get "": unsupported protocol scheme ""
```

File: resources/run_adl.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+# 4. set relevant environment variables
+export GOPRIVATE = github.com/10gen
+export MONGOHOUSE_ENVIRONMENT = "local"
+export MONGOHOUSE_MQLRUN = "$(pwd)/artifacts/mqlrun"
+export LIBRARY_PATH="$(pwd)/artifacts"
```
Comment (bucaojit) [inline resources/run_adl.sh:20]:
```suggestion
export GOPRIVATE=github.com/10gen
export MONGOHOUSE_ENVIRONMENT="local"
export MONGOHOUSE_MQLRUN="$(pwd)/artifacts/mqlrun"
export LIBRARY_PATH="$(pwd)/artifacts"
```

File: resources/run_adl.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+# 5. download mongod
+# TODO - what platform? or should it be all platforms? parameterize this
+#   - this is an example platform
+curl -O https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-rhel70-5.0.3.tgz
```
Comment (bucaojit) [inline resources/run_adl.sh:25]: For Ubuntu1804
```
curl -O https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-5.0.4.tgz
```
The team suggested that Mac and Linux be supported.

**Comment by mattChiaravalloti** [comment]
closed in favor of #63

**Pull request closed without merging**